src/main/resources/model/bibframe_authority.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "bibframe-authority",
+  "eventBodyJavaClass": "org.folio.search.domain.dto.BibframeAuthority",
+  "languageSourcePaths": ["$.languages"],
+  "fields": {
+    "id": {
+      "index": "keyword"
+    },
+    "label": {
+      "index": "whitespace"
+    },
+    "type": {
+      "index": "keyword"
+    },
+    "identifiers": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "index": "whitespace"
+        },
+        "type": {
+          "index": "whitespace"
+        }
+      }
+    }
+  },
+  "searchFields": {
+    "label": {
+      "type": "search",
+      "index": "multilang",
+      "processor": "bibframeAuthorityLabelProcessor"
+    },
```

Comment:
I think it will override the `label` field in the `fields` block as the names are the same. Maybe it makes sense to set index=multilang for label in fields block.

Reply:
You still have same field "label" in both searchFields and Fields. If there is no special processing needed for the field, then I suggest to keep only one that is in Fields.